content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -259,36 +315,42 @@ beforeEach(() => {
 #### Execution
 
 Cypress executes the support file before the spec file. For example when you
-click on a test file named `spec-a.js` via
+click on a test file named `spec-a.cy.js` via
 [cypress open](/guides/guides/command-line#cypress-open), then the Cypress App
 executes the files in the following order:
 
-```html
-<!-- bundled support file -->
-<script src="support/index.js"></script>
-<!-- bundled spec file -->
-<script src="integration/spec-a.js"></script>
-```
+**e2e example:**
+
+1. `support/e2e.js // your support file`
+2. `e2e/spec-a.cy.js // your spec file`
+
+**component example:**
+
+1. `support/component.js // your support file`
+2. `components/Button/Button.cy.js // your spec file`
 
 The same happens when using the
 [cypress run](/guides/guides/command-line#cypress-run) command: a new browser
 window is opened for each support and spec file pair.
 
 But when you click on "Run all specs" button after
```

Comment:
Can you find out if Cypress still has a "Run all specs" button? I didn't see it when using a 10.0-release build just now. I want to make sure this section is still relevant.

Reply:
This commit removes these examples: https://github.com/cypress-io/cypress-documentation/pull/4328/commits/10df3603af19ddce991e37e42a8fcba215233d8f

I also created a follow up issue to remove other instances in the docs where we reference `Runs All`: https://cypress-io.atlassian.net/browse/UDX-118